_data/publications.yml

```diff
@@ -142,10 +142,12 @@
   slug: cases_lead_hospitalizations
   authors: Reich NG, Wang Y, Burns M, Ergas R, Cramer EY, Ray EL
   preprint: https://www.medrxiv.org/content/10.1101/2023.03.08.23286582v1
-  pdf: https://www.medrxiv.org/content/10.1101/2023.03.08.23286582v1.full.pdf
+  pdf: https://pdf.sciencedirectassets.com/277421/1-s2.0-S1755436523X00048/1-s2.0-S1755436523000646/main.pdf?X-Amz-Security-Token=IQoJb3JpZ2luX2VjEKb%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FwEaCXVzLWVhc3QtMSJHMEUCIHA1JlBLaCRW0exyLzLDoL1DQhwbBxLdniQF1JtSkVY2AiEAjtoXCSK622tIJGn2LHZYFwPfpPYOuyudAoPWHHyHMp0quwUI7v%2F%2F%2F%2F%2F%2F%2F%2F%2F%2FARAFGgwwNTkwMDM1NDY4NjUiDD56Mc9C4sGIuASsfyqPBW9nQ1XPMsFVwfpqnWzR1KDupm6ydYfJln4099FIho8Yz433Lqw6aI%2BM09vvH140qPin%2BObKvMWxOGZISzolLJNlsX%2FkUl7oMztI1eEebhEVzVQvZtGcyW6HeR1P6tRCP4JKtRrj9tok92G6ntwNCRlKONY8WyhafBVY0y6V3qrMpngj9%2BJcVGauuM5fxTCnRMGoyV51zMh4gtcy3EM552VVObeOQErwpyMQ%2BTc%2FDqYQga6%2BfnuqEi842ztpmWUp70fSa4nY4ENfrlH3LhYE29Sb8n8eOz9NIbsmnfqP73KFYVRElfsrF0el8mhS8ipoIYoLLGFOLKBPgYW6FqOyFPklfrBW67RcHc05qHlHJm%2Fg1kHmdhG340MteEYAk%2BICWqKcLB78kjLSCGcCyIX5FVYnyMu%2BiAGKUlcuS3IqIQTZbrMjdSJCHLUCKOtwfjPQARz5JGvNpVGw4Xlv7m5PSMC%2FJETqeo5u9JodkKkqMvaq4ZUfTCiSNiMQGapWgFaxH2DKxX2AI4ZO%2FJn0XN5gPNgpIGupOlBmn91j4HwZtv0yrw488XnBKVdNqcSSdRyInAmOqnu%2Fc938WWPgr%2BY0qBhqZISIa3KLL1XEAM56oHuYcLolkxKI6DbdOPxGItlMTLk%2FlpSa6sNx9dlrBk8gqm%2FvpdvOk3vhoWpUYLnnGaPrmwFMtc5l3UgMr%2BnvkUin0zZPXmqefyMuZgQhafkAgeBC%2FCBp6aX2nzMqviURIx6ydMvNEmyfK1kwAbomi%2BSGEDABwVlA67HrH7hjFWf6%2FqXxX2Ifqa%2B2xpl5Mayz6V%2Bw4p27rFmt0pNn2wXWKawFYQQ4GueOAoct3tuQKzBCpb5CtJnzFe5j%2BVvybiowAXYwk7PtqgY6sQF34Fi9GmjPsJe9zOgOwPtTmYkkxB7yF7TSCapSzCsieNfl%2Fz7Z1fjOsNCsJYMZbTcL%2F8HydxGoSkEe%2Bz%2BWWusA%2FRbqqyGCUozKbMnFkj6U01VMZ3%2BecuPso469FiKsyfuScv95NbUlcg3O%2FjRSNutNKZTklWL2srMgcg30QG9TbGd8lfy2zuWMA%2FDgnOSLqdJtIaO%2Fz177hNo1OG156W671gLxAtXsMJqA6RdTBXpCdbQ%3D&X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20231120T134641Z&X-Amz-SignedHeaders=host&X-Amz-Expires=300&X-Amz-Credential=ASIAQ3PHCVTYYHLRCROO%2F20231120%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=88e89f8c2084eae2001e9dbfae7e4d118149df2b63aa650dee0559f9f713172d&hash=903407d8cfa56e56713426f60ffe777d95cdffec7e181b94e4440fc2c55a256c&host=68042c943591013ac2b2430a89b270f6af2c76d8dfd086a07176afe7c76c2c61&pii=S1755436523000646&tid=spdf-776eef55-3d9c-4873-800f-88868580d069&sid=0e2d6c1c8920104c11999b65e482402a35d3gxrqa&type=client&tsoh=d3d3LnNjaWVuY2VkaXJlY3QuY29t&ua=13115651565755530959&rr=829123f61c59304d&cc=us
```

Comment:
I think this link should be changed to this:
https://www.sciencedirect.com/science/article/pii/S1755436523000646/pdfft?md5=501f8c5a916c41a194955789cddb3e5e&pid=1-s2.0-S1755436523000646-main.pdf

I got this link by right clicking on the "view PDF" link on the page and then copying the link. When I load this link it brings me to a PDF, but when I load the link in the file here I just get to the main paper page, without a pdf.